chapters/memory-security/reverse-engineering/guides/ghidra-tutorial/support/crackme.c

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include <string.h>
+
+void usage(void)
+{
+	printf("Usage: ./crackme <password>\n");
+	exit(0);
+}
+
+int main(int argc, char *argv[])
+{
+	if (argc == 2) {
+		uint32_t len = strlen(argv[1]);
+
+		if (len == 8) {
+			if (*(argv[1] + 3) == 'E') {
+				printf("Good job!\n");
+				exit(0);
+			}
+		}
+		printf("Try again!\n");
+	} else {
+		usage();
+	}
```

Comment:
```suggestion
	} else
		usage();
```

Reply:
I recall te linter requiring that as long as the main branch of the `if` statement has brackets, the `else` branch should have them too

Reply:
Fuck, you're right. Sorry.